packages/consumer/src/methods.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  UserAccountEvent,
+  AbortedEvent,
+  TransactionEvent,
+} from "@chainsafe/eth-sso-common";
+import type { Transaction, UserAccount } from "@chainsafe/eth-sso-common";
+import { handleRedirect, windowOpen } from "./utils";
+
+export async function authenticate(
+  providerUrl: string,
+  redirectUrl: string,
+  chainId: string,
```

Comment:
shoul.d we validated that this is caip chainId and redirectUrl is proper url?

Reply:
Let's try to pass POC/MVP and then see how to improve DX and the reliability of tools